### PR TITLE
Added 3 new keyboards

### PR DESCRIPTION
**Pull request by DeskDaily**
## Description

 Added 4 folders for 4 new keyboards

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/nightly_boards/alter_lite/config.h [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xD812
+#define PRODUCT_ID 0x0013
+#define DEVICE_VER 0x0001
+#define MANUFACTURER DeskDaily
+#define PRODUCT Alter Lite
+#define DESCRIPTION An Alice layout keyboard
```
Comment (drashna) [inline keyboards/nightly_boards/alter_lite/config.h:25]:
```suggestion
```

File: keyboards/nightly_boards/alter_lite/config.h [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+
+
+/* disable these deprecated features by default */
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
```
Comment (drashna) [inline keyboards/nightly_boards/alter_lite/config.h:68]:
```suggestion
```

File: keyboards/nightly_boards/alter_lite/rules.mk [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+#
+BOOTMAGIC_ENABLE = yes        # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes         # Mouse keys
+EXTRAKEY_ENABLE = yes         # Audio control and System control
+CONSOLE_ENABLE = no           # Console for debug
+COMMAND_ENABLE = no           # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no         # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes             # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/nightly_boards/alter_lite/rules.mk:18]: Please remove the `SLEEP_LED_ENABLE` setting, as it's not really used, and remove the comments for both it and NKRO.  
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

File: keyboards/nightly_boards/conde60/config.h [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+/* define if matrix has ghost (lacks anti-ghosting diodes) */
+//#define MATRIX_HAS_GHOST
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
```
Comment (drashna) [inline keyboards/nightly_boards/conde60/config.h:69]:
```suggestion
```

File: keyboards/nightly_boards/conde60/rules.mk [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+#
+BOOTMAGIC_ENABLE = yes        # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes         # Mouse keys
+EXTRAKEY_ENABLE = yes         # Audio control and System control
+CONSOLE_ENABLE = no           # Console for debug
+COMMAND_ENABLE = no           # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no         # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes             # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/nightly_boards/conde60/rules.mk:18]: Please remove the `SLEEP_LED_ENABLE` setting, as it's not really used, and remove the comments for both it and NKRO.  
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

File: keyboards/nightly_boards/octopadplus/config.h [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+#define MATRIX_COL_PINS { E6, F5, C7, D7, F4, D2 }
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+#define RGB_DI_PIN F7
+#ifdef RGB_DI_PIN
+#define RGBLED_NUM 8
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/nightly_boards/octopadplus/config.h:53]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/nightly_boards/octopadplus/octopadplus.c [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+void matrix_scan_kb(void) {
+    encoder_action_unregister();
+    matrix_scan_user();
+}
+
+bool encoder_update_kb(uint8_t index, bool clockwise) {
+//    if (!encoder_update_user(index, clockwise)) return false;
+    encoder_action_register(index, clockwise);
+    return true;
+};
```
Comment (drashna) [inline keyboards/nightly_boards/octopadplus/octopadplus.c:28]: Please move all of this stuff to the via keymap only. 

Also, you may want to use encoder map instead. 
https://docs.qmk.fm/#/feature_encoders?id=encoder-map

File: keyboards/nightly_boards/octopadplus/octopadplus.h [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+#include "encoder_action.h"
```
Comment (drashna) [inline keyboards/nightly_boards/octopadplus/octopadplus.h:20]:
```suggestion
```

File: keyboards/nightly_boards/octopadplus/octopadplus.h [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT( \
+    E00A, K04,   E00B,  E01A, K15,  E01B, \
```
Comment (drashna) [inline keyboards/nightly_boards/octopadplus/octopadplus.h:31]: the default layout should not include encoder support, like this.

File: keyboards/nightly_boards/paraluman/config.h [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+/* define if matrix has ghost (lacks anti-ghosting diodes) */
+//#define MATRIX_HAS_GHOST
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
```
Comment (drashna) [inline keyboards/nightly_boards/paraluman/config.h:62]:
```suggestion
```

File: keyboards/nightly_boards/paraluman/rules.mk [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+#
+BOOTMAGIC_ENABLE = yes        # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes         # Mouse keys
+EXTRAKEY_ENABLE = yes         # Audio control and System control
+CONSOLE_ENABLE = no           # Console for debug
+COMMAND_ENABLE = no           # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no         # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes             # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/nightly_boards/paraluman/rules.mk:18]: Please remove the `SLEEP_LED_ENABLE` setting, as it's not really used, and remove the comments for both it and NKRO.  
```suggestion
NKRO_ENABLE = yes           # Enable N-Key Rollover
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/nightly_boards/paraluman/rules.mk (hunk 11 above)
Comment (DeskDaily) [reply keyboards/nightly_boards/paraluman/rules.mk:18]: Thank you @drashna Will edit these

**Comment by DeskDaily** [comment]
I removed the Octopad+ folder. Will edit that in a separate pull request instead @drashna

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]
In future, it'll probably be simpler to raise separate PRs for each keyboard.

File: keyboards/nightly_boards/paraluman/info.json [hunk 12]
```diff
@@ -0,0 +1,150 @@
+{
+    "keyboard_name": "Paraluman", 
+    "url": "", 
+    "maintainer": "DeskDaily", 
+    "width": 17.25, 
+    "height": 5,
```
Comment (fauxpark) [inline keyboards/nightly_boards/paraluman/info.json:6]: These need to be removed.

File: keyboards/nightly_boards/conde60/readme.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
+# DeskDaily Conde60
+
+![Conde60](https://media.discordapp.net/attachments/755074685862936686/982918555882360852/unknown.png?width=853&height=396)
```
Comment (fauxpark) [inline keyboards/nightly_boards/conde60/readme.md:3]: This should be an imgur link.

File: keyboards/nightly_boards/alter_lite/readme.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
+#DeskDaily Alter
+
+![Alter Lite](https://media.discordapp.net/attachments/946989936941346857/946992282576519279/PXL_20220226_044706997.PORTRAIT2.jpg?width=704&height=396)
```
Comment (fauxpark) [inline keyboards/nightly_boards/alter_lite/readme.md:3]: As above.

File: keyboards/nightly_boards/alter_lite/info.json [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+			{"x":6.75, "y":0.24999999999999997}, 
+			{"x":7.75, "y":0.24999999999999997}, 
+			{"x":11.25, "y":0.24999999999999997}, 
+			{"x":12.25, "y":0.24999999999999997}, 
+			{"x":13.25, "y":0.24999999999999997}, 
+			{"x":14.25, "y":0.24999999999999997}, 
+			{"x":15.25, "y":0.24999999999999997}, 
+			{"x":16.25, "y":0.24999999999999997}, 
+			{"x":17.25, "y":0.24999999999999997}, 
+			{"x":18.25, "y":0.24999999999999997},
```
Comment (fauxpark) [inline keyboards/nightly_boards/alter_lite/info.json:25]: This is not correct.

File: keyboards/nightly_boards/alter_lite/info.json [hunk 16]
```diff
@@ -0,0 +1,78 @@
+{
+    "keyboard_name": "Alter",
+    "url": "",
+    "maintainer": "DeskDaily",
+    "width": 20,
+    "height": 5.25,
```
Comment (fauxpark) [inline keyboards/nightly_boards/alter_lite/info.json:6]: As above.

**Pull request merged**